### PR TITLE
Add Moddermore

### PR DESCRIPTION
**Pull request by ryanccn**
https://moddermore.vercel.app

https://github.com/ryanccn/moddermore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,7 @@ Feel free to add your own project(s) -- just fork and make a pull request! We su
 - **[RozeFound/mmc-export](https://github.com/RozeFound/mmc-export)** - A tool for exporting a MultiMC modpack to other formats
 - **[shap-po/selene-modpacker](https://github.com/shap-po/selene-modpacker)** - A modpack creation browser extension allowing you to save mods from Modrinth and CurseForge to collections
 - **[Tom-The-Geek/pack-it](https://github.com/Tom-The-Geek/pack-it)** - A CLI tool for managing Minecraft mods
+- **[Moddermore](https://moddermore.vercel.app?ref=modrinthawesome)** - A web app for creating public lists of mods exportable to `mrpack`s
```
Comment (ExoPlant) [inline README.md:97]: smh ref link

Comment (wafflecoffee) [inline README.md:97]: Please put it in alphabetical order and make it the same style as the frontend section with the GitHub repo first and the website link after.

**Review by wafflecoffee — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ryanccn) [reply README.md]: fixed

Comment (ryanccn) [reply README.md]: `github.com` isn't all that informative 🫠

**Review by wafflecoffee — APPROVED** [review]

**Pull request merged**